<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Publications | Etienne  Sollier</title>
    <meta name="author" content="Etienne  Sollier">
    <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
">
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Bootstrap Table -->
    <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.3/dist/bootstrap-table.min.css">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700%7CRoboto+Slab:100,300,400,500,700%7CMaterial+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    

    <!-- Styles -->
    
    <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;ES&lt;/text&gt;&lt;/svg&gt;">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://e-sollier.github.io/publications/">

    <!-- Dark Mode -->
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Etienne </span>Sollier</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">About</a>
              </li>
              
              <!-- Blog -->
              <li class="nav-item ">
                <a class="nav-link" href="/blog/">Blog</a>
              </li>

              <!-- Other pages -->
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">Publications<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/cv/">CV</a>
              </li>
            </ul>
          </div>
        </div>
      </nav>

    </header>


    <!-- Content -->
    <div class="container mt-5">
      
        <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">Publications</h1>
            <p class="post-description"></p>
          </header>

          <article>
            <!-- _pages/publications.md -->
<div class="publications">
  <h2 class="year">2023</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge">biorxiv</abbr></div>

        <!-- Entry bib key -->
        <div id="sollier_joint_2023" class="col-sm-8">
        <!-- Title -->
        <div class="title">Joint copy number and mutation phylogeny reconstruction from single-cell amplicon sequencing data</div>
        <!-- Author -->
        <div class="author">
        

        <b><u>Etienne Sollier</u></b>, Jack Kuipers, Koichi Takahashi, Niko Beerenwinkel, and Katharina Jahn</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>biorxiv</em>, Mar 2023
        </div>
        <div class="periodical">
        <a href="https://www.biorxiv.org/content/10.1101/2022.01.06.475205v3" rel="external nofollow noopener" target="_blank">https://www.biorxiv.org/content/10.1101/2022.01.06.475205v3</a>
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>Reconstructing the history of somatic DNA alterations can help understand the evolution of a tumor and predict its resistance to treatment. Single-cell DNA sequencing (scDNAseq) can be used to investigate clonal heterogeneity and to inform phylogeny reconstruction. However, most existing phylogenetic methods for scDNAseq data are designed either for single nucleotide variants (SNVs) or for large copy number alterations (CNAs), or are not applicable to targeted sequencing. Here, we develop COMPASS, a computational method for inferring the joint phylogeny of SNVs and CNAs from targeted scDNAseq data. We evaluate COMPASS on simulated data and apply it to several datasets including a cohort of 123 patients with acute myeloid leukemia. COMPASS detected clonal CNAs that could be orthogonally validated with bulk data, in addition to subclonal ones that require single-cell resolution, some of which point toward convergent evolution.</p>
          </div>
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge"><a href="https://www.nature.com/leu/" rel="external nofollow noopener" target="_blank">Leukemia</a></abbr></div>

        <!-- Entry bib key -->
        <div id="weichenhan_translocation_2023" class="col-sm-8">
        <!-- Title -->
        <div class="title">Translocation t(6;7) in AML-M4 cell line GDM-1 results in MNX1 activation through enhancer-hijacking</div>
        <!-- Author -->
        <div class="author">
        

        Dieter Weichenhan, Anna Riedel, Charlotte Meinen, Alisa Basic, Reka Toth, Marion Bähr, Pavlo Lutsik, Joschka Hey, <b><u>Etienne Sollier</u></b>, Umut H. Toprak, Simge Kelekçi, Yu-Yu Lin, Mariam Hakobyan, Aurore Touzart, Ashish Goyal, Justyna A. Wierzbinska, Matthias Schlesner, Frank Westermann, Daniel B. Lipka, and Christoph Plass</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Leukemia</em>, May 2023
        </div>
        <div class="periodical">
        <a href="https://www.nature.com/articles/s41375-023-01865-5" rel="external nofollow noopener" target="_blank">https://www.nature.com/articles/s41375-023-01865-5</a>
        </div>

          <!-- Links/Buttons -->
          <div class="links">
          </div>
          
          <div class="badges">
          </div>

          
        </div>
      </div>
</li>
<li>
<!-- _layouts/bib.html -->
      <div class="row">
        <div class="col-sm-2 abbr"><abbr class="badge"><a href="https://haematologica.org/index" rel="external nofollow noopener" target="_blank">Haematologica</a></abbr></div>

        <!-- Entry bib key -->
        <div id="smith_tal1_2023" class="col-sm-8">
        <!-- Title -->
        <div class="title">TAL1 activation in T-cell acute lymphoblastic leukemia: a novel oncogenic 3’ neo-enhancer</div>
        <!-- Author -->
        <div class="author">
        

        Charlotte Smith, Ashish Goyal, Dieter Weichenhan, Eric Allemand, Anand Mayakonda, Umut Toprak, Anna Riedel, Estelle Balducci, Manisha Manojkumar, Anastasija Pejkovska, Oliver Mücke, <b><u>Etienne Sollier</u></b>, Ali Bakr, Kersten Breuer, Pavlo Lutsik, Olivier Hermine, Salvatore Spicuglia, Vahid Asnafi, Christoph Plass, and Aurore Touzart</div>

        <!-- Journal/Book title and date -->
        
        
        <div class="periodical">
          <em>Haematologica</em>, Jan 2023
        </div>
        <div class="periodical">
        <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10153542/" rel="external nofollow noopener" target="_blank">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10153542/</a>
        </div>

          <!-- Links/Buttons -->
          <div class="links">
            <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a>
          </div>
          
          <div class="badges">
          </div>

          <!-- Hidden abstract block -->
          <div class="abstract hidden">
            <p>T-cell acute lymphocytic leukemia protein 1 (TAL1) is one of the most frequently deregulated oncogenes in T-cell acute lymphoblastic leukemia (T-ALL). Its deregulation can occur through diverse cis-alterations, including SIL-TAL1 microdeletions, translocations with T-cell Receptor loci, and more recently described upstream intergenic non-coding mutations. These mutations consist of recurrent focal microinsertions that create an oncogenic neo-enhancer accompanied by activating epigenetic marks. This observation laid the groundwork for an innovative paradigm concerning the activation of proto-oncogenes via genomic alterations of non-coding intergenic regions. However, for the majority of T-ALL expressing TAL1 (TAL1+), the deregulation mechanism remains ’unresolved’. We took advantage of H3K27ac and H3K4me3 chromatin immunoprecipitation sequencing data of eight cases of T-ALL, including five TAL1+ cases. We identified a putative novel oncogenic neo-enhancer downstream of TAL1 in an unresolved monoallelic TAL1+ case. A rare but recurrent somatic heterozygous microinsertion within this region creates a de novo binding site for MYB transcription factor. Here we demonstrate that this mutation leads to increased enhancer activity, gain of active epigenetic marks, and TAL1 activation via recruitment of MYB. These results highlight the diversity of non-coding mutations that can drive oncogene activation.</p>
          </div>
        </div>
      </div>
</li>
</ol>

  <h2 class="year">2022</h2>
  <ol class="bibliography"></ol>


</div>

          </article>

        </div>

      
    </div>

    <!-- Footer -->
    <!--    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        &copy; Copyright 2023 Etienne  Sollier. 
      </div>
    </footer> -->

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script>

  <!-- Bootstrap Table -->
  <script defer src="https://unpkg.com/bootstrap-table@1.21.3/dist/bootstrap-table.min.js"></script>

  <!-- Load Common JS -->
  <script src="/assets/js/no_defer.js"></script>
  <script defer src="/assets/js/common.js"></script>
  <script defer src="/assets/js/copy_code.js" type="text/javascript"></script>

    

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
    

  </body>
</html>
